wgpu-types/src/lib.rs

```diff
@@ -1414,6 +1414,8 @@ pub struct AdapterInfo {
     pub driver_info: String,
     /// Backend used for device
     pub backend: Backend,
+    /// Supports [`TextureUsages::TRANSIENT`]
+    pub supports_transient: bool,
```

Comment:
```suggestion
    pub native_transient_texture: bool,
```

Really not set on this name, but what I'm trying to avoid is the implication that TRANSIENT is not allowed

Reply:
This still needs to be updated

Reply:
Oops, been pretty busy recently. I should be able to finish the rest of the changes this weekend. Sorry for the wait.

Reply:
No worries or rush! Just re-request a review from me whenever you're ready!

Reply:
Changed it to `transient_saves_memory`, please review if that name looks good or if I should change it to anything else!